manim/utils/family.py

```diff
@@ -28,6 +28,9 @@ def extract_mobject_family_members(
         method = Mobject.family_members_with_points
     else:
         method = Mobject.get_family
+    extracted_mobjects = remove_list_redundancies(
+        list(it.chain(*[method(m) for m in mobjects]))
+    )
```

Comment:
**Comment** At some point all the iterable handling in manim sutff should definitely be changed, as this is .. hmm .. not pretty

Reply:
Agreed, we should refactor this. Stuff concerning families is really weird.